src/stores/datStore.ts

```diff
@@ -291,14 +292,16 @@ export default class DatStore {
             buyEvent.blockNumber
         );
 
+        const chainId = this.rootStore.providerStore.getActiveWeb3React().chainId;
+        console.log(chainId);
```

Comment:
Should probably remove the log statements

Reply:
removed